sdk/keyvault/Azure.ResourceManager.KeyVault/Directory.Build.props

```diff
@@ -1,4 +1,8 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="$(MSBuildProjectName.EndsWith('.Samples'))">
+    <IsSampleProject>true</IsSampleProject>
```

Comment:
Do we need to add the property `IsSampleProject` here? I don't find it affecting anything.

Reply:
removed